src/jv_aux.c

```diff
@@ -222,13 +222,17 @@ jv jv_dels(jv t, jv keys) {
     // no change
```

Comment:
Huh.  jv_dels() should be static or else it should have a prototype in jv.h.  I don't know that I care which (I see no reason that it wouldn't be stable, if we add the prototype).